src/Xamarin.Android.Build.Tasks/Xamarin.Android.Bindings.targets

```diff
@@ -366,16 +366,20 @@ Copyright (C) 2012 Xamarin Inc. All rights reserved.
       <Output TaskParameter="Include" ItemName="ReferenceJar" />
     </CreateItem>
 
+    <ItemGroup>
+      <_AndroidDocumentationPath Include="$(AndroidDocumentationPath)" />
```

Comment:
I think this should be `@(AndroidDocumentationPath)`  (item group), not `$(AndroidDocumentationPath)` (property).